testing/environments/docker/elasticsearch_kerberos/healthcheck.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
 
 # check if service principal is OK
-KRB5_CONFIG=/etc/krb5.conf \
-    kinit -k -t /etc/HTTP_elasticsearch_kerberos.elastic.keytab HTTP/elasticsearch_kerberos.elastic@ELASTIC
-
+export KRB5_CONFIG=/etc/krb5.conf 
+kinit -k -t /etc/HTTP_localhost.keytab HTTP/localhost@$REALM
 
 # check if beats user can connect
-echo testing | KRB5_CONFIG=/etc/krb5.conf kinit beats@ELASTIC
+kinit beats@$REALM
 klist
-curl --negotiate -u : -XGET http://elasticsearch_kerberos.elastic:9200/
+
+curl --negotiate -u : -XGET http://localhost:9200/
```

Comment:
another nit but is there a good reason to switch to localhost from `elasticsearch_kerberos.local`? The second one is more grep-able and easier to understand.